hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -246,7 +248,7 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
 
   private DBStore store;
 
-  private final IOzoneManagerLock lock;
+  private IOzoneManagerLock lock;
```

Comment:
why not final?

Reply:
Because I added a new `OmMetadataManagerImpl` constructor and moved common logic to `init()` method where `lock` is then assigned.

Reply:
There are 40-ish places that calls the old constructor. I could fix that instead.

Reply:
> There are 40-ish places that calls the old constructor. I could fix that instead.

Not necessary, I just wanted to understand.